### ISSUE TITLE
Fetching, archiving files (No file transfer via slack policy) 

### ISSUE THREAD

**Issue by cgpu** [TheJacksonLaboratory/sbas#51]
Adding this here to keep in mind and pinning for easy access:

## tl;dr - `wget` able url for a tar.gz for all files Anne mentions above:

```

https://github.com/lifebit-ai/lifebitCloudOSDREgtex/releases/download/fig1c_archive/datafig1c18022020.tar.gz

```

## Overview of no sending files policy 👮‍♀ 

@karleg @adeslatt let's avoid sending files because this will have as a non-desired side effect local files making an appearance in subsequent scripts as the very first input.

I archived everything Anne worked on today here:

https://github.com/lifebit-ai/lifebitCloudOSDREgtex/releases/tag/fig1c_archive

The idea is to start adding releases in this repo as well, I will implement the automation myself.

For now let's agree that we :warning: **don't send** files with 

- slack 
- mail
- etc

Let's use `wget` so that 
- we don't include any local paths as the starting point of input data in the scripts
- be able to verify with the `sha256sum` that the files between steps are correct

## How to fetch the data from `GitHub Releases` with `wget`
 Find the archive here: 
https://github.com/lifebit-ai/lifebitCloudOSDREgtex/releases/tag/fig1c_archive

![image](https://user-images.githubusercontent.com/38183826/74790158-b9032680-52ae-11ea-804e-9ff1ace59264.png)

## 1. `wget`:  Grab the url link from GitHub Releases

```bash
wget https://github.com/lifebit-ai/lifebitCloudOSDREgtex/releases/download/fig1c_archive/figure1c_heatmap.rds
```

## 2. `sha256sum`: Verify origin and destination files are identical

```bash
sha256sum figure1c_heatmap.rds
```
See gif:

![4WH2ZmQPJK](https://user-images.githubusercontent.com/38183826/74790322-29aa4300-52af-11ea-8a62-bce281122931.gif)

I think this will help me implementing all this in NextFlow as well, as I have to be mindful of all input outputs.

Let me know if this sounds good and we commit to fetch intermediate files within the script.
